models/citizen.ts

```diff
@@ -80,6 +105,12 @@ export class Citizen {
   })
   payment_instrument_update_user?: string;
 
+  @ViewColumn({
+    // TODO this name contains an error, it should be pay_istr_enable_b
```

Comment:
why this _TODO_?

Reply:
there is a typo on the view, `pay_istr_update_enable_b`  should be `pay_istr_enable_b`. Now I don't know when will be fixed on the view